tests/tests_unit/test_config.py

```diff
@@ -0,0 +1,164 @@
+from pathlib import Path
+from unittest.mock import MagicMock, mock_open
+
+from cognite.neat._config import (
+    ModelingConfig,
+    NeatConfig,
+    ValidationConfig,
+)
+
+
+class TestNeatConfig:
+    """Test suite for NeatConfig - governance profiles and configuration loading."""
+
+    def test_default_initialization(self) -> None:
+        """Test default NeatConfig initialization."""
+        config = NeatConfig()
+
+        assert config.profile == "legacy-additive"
+        assert config.modeling.mode == "additive"
+        assert config.validation.exclude == [
+            "NEAT-DMS-AI-READINESS-*",
+            "NEAT-DMS-CONNECTIONS-002",
+            "NEAT-DMS-CONNECTIONS-REVERSE-007",
+            "NEAT-DMS-CONNECTIONS-REVERSE-008",
+            "NEAT-DMS-CONSISTENCY-001",
+        ]
```

Comment:
I think, if possible, we avoid having multiple asserts. If the default is wrong and the tests fails. Frist, you will have to fix the profile, rerun tests, discover that the mode is wrong, fix that, rerun test, and then fix exclude. By doing a single assert that capture all of it, you get a nice error telling you all that is wrong, you fix it, and you are done.

```suggestion
    EXPECTED_DEFAULT_CONFIG = NeatConfig(
        profile="legacy-additive",
        modeling=ModelingConfig(mode="additive"),
        validation=ValidationConfig(
            exclude=[
                "NEAT-DMS-AI-READINESS-*",
                "NEAT-DMS-CONNECTIONS-002",
                "NEAT-DMS-CONNECTIONS-REVERSE-007",
                "NEAT-DMS-CONNECTIONS-REVERSE-008",
                "NEAT-DMS-CONSISTENCY-001",
        ])
    )

    def test_default_initialization(self) -> None:
        """Test default NeatConfig initialization."""
        config = NeatConfig()

        assert config == self.EXPECTED_DEFAULT_CONFIG
```

Reply:
There is also something strange about this tests. If I modify it to the following, it still passes.
```python
    EXPECTED_DEFAULT_CONFIG = NeatConfig(
        profile="legacy-additive",
        modeling=ModelingConfig(mode="additive"),
        validation=ValidationConfig(exclude=["NEAT-DMS-AI-READINESS-*"])
    )

    def test_default_initialization(self) -> None:
        """Test default NeatConfig initialization."""
        config = NeatConfig()

        assert config == self.EXPECTED_DEFAULT_CONFIG
```

Reply:
Tests are now heavily simplified and more readable